runtime/src/iree-amd-aie/runtime/CMakeLists.txt

```diff
@@ -0,0 +1,144 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions. See
+# https://llvm.org/LICENSE.txt for license information. SPDX-License-Identifier:
+# Apache-2.0 WITH LLVM-exception
+
+# ##############################################################################
+# requirements
+# ##############################################################################
+
+if(NOT ${CMAKE_SIZEOF_VOID_P} EQUAL 8)
+  # if you open up the wrong developer shell in windows...
+  message(
+    FATAL_ERROR
+      "Something has gone terribly wrong: CMAKE_SIZEOF_VOID_P=${CMAKE_SIZEOF_VOID_P}"
+  )
+endif()
+# can delete after this is all worked out
```

Comment:
Bootgen was retrofitted for our (AIE) use case; otherwise it is used for distributing RTL IP (netlists or whatever) for FPGA customers that aim to sell their own designs. In this capacity it provides the ability to sign the artifact. For us it's useless functionality (especially since xclbin signature checking was turned off earlier this year) but I haven't yet unwound their code enough to decouple from that functionality. But it is absolutely part of planned for the "reducing frictions" part of the roadmap (I just forgot to put it on there).

Reply:
Sounds like a good plan, thanks. Answers the OpenSSL question too.